### DISCUSSION TITLE
The future of Covenant

### DISCUSSION THREAD

**Discussion by madalin-dogaru** [cobbr/Covenant#295] [General]
Hello @cobbr , 

First, let me congratulate you on what you are doing with Covenant, this is amazing work!
I have no experience with C#/.NET but but I've found Covenant after joining the RTO course from zeropointsecurity and I'm starting to get attracted by both Covenant and C#/.NET. 

Although, I can fork this project and work on it, I would really like to work on the current project after I finish RTO but I'm just curious if you still have plans for Covenant to develop it or it was just a "test project" ? 

From what I can see its being developed but I just wanted to see your vision to the project before I start helping. 

Madalin

**Comment by cobbr** [comment]
Hey @john-connort8001 , I don't have any plans to stop working on the project. There's been ~100 commits over the last month if that helps give you any idea.

**Reply by madalin-dogaru** [reply]
Yep, its active just wanted to see what you think. These are great news.

**Reply by cobbr** [reply]
Sounds great!

**Comment by tarayman813-hub** [comment]
اختراق الهاتف
